bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -86,6 +87,43 @@ async def headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    @staticmethod
+    def _make_query_body(
+            query: str,
+            write_disposition: Disposition,
+            use_query_cache: bool,
+            dry_run: bool, use_legacy_sql: bool,
+            destination_table: Optional[Dict[str, object]]) -> Dict[str, Any]:
```

Comment:
Might make more sense to have this method accept an `Optional[Table]` directly, since this method seems to be responsible for doing all the jsonification -- as you currently have it, you need the `insert_via_query` method to have some understanding of the payload format, which is non-ideal since that spreads this info across multiple places.

Reply:
yes, overthought myself. fixed now